openapi.yaml

```diff
@@ -6,6 +6,24 @@ info:
     title: ""
     version: 0.0.1
 paths:
+    /api/authz/v1/check:
```

Comment:
How much do we care about using RESTful verbs in a meaningful way?
i.e. Some sources (e.g. https://stackoverflow.com/questions/2001773/understanding-rest-verbs-error-codes-and-authentication) say that POST would be CREATE, whereas GET would retrieve (the permissionship). 

We could just decide that this endpoint will not be RESTful, but we did go down the route of trying to use the verbs logically in the relationships spec.

Reply:
Yes, Authzed follow the convention of using post with the check endpoints: https://petstore.swagger.io/?url=https://raw.githubusercontent.com/authzed/authzed-go/main/proto/apidocs.swagger.json#/PermissionsService/PermissionsService_CheckPermission

Reply:
+1 to POST, makes it much easier for clients and keeps the URL/Endpoint simple and its inline with underlying spiceDB "POST" for the check endpoint

Reply:
We can discuss it and finalize it.